### ISSUE TITLE
Change of label: Macedonia to North Macedonia

### ISSUE THREAD

**Issue by simboss** [geosolutions-it/eumetsat-EOWS#418]
Long story short, the OSM data in GPKG that we delivered to EMSA need a minor update to go from Macedonia to North Macedonia.

@aaime any hints on how we can do that without redeploying or such? I mean, can we run a SQL command without having to redeploy the whole GPKG?

https://cgi-pathfinders.atlassian.net/browse/IMGMT-402

**Comment by aaime** [comment]
There are two paths to the update, one relatively easy, one a bit more annoying.

If all we care about is what's in the WMS/WMTS tile, then we just need to update the ne_10m_admin_0_countries_points, in particular, the LONG_NAME attribute. This cannot be done with sqlite3 library, as I thought, because the triggers in a GeoPackage need a few spatial functions not provided by sqlite3. However, ogr has them, so the following works:

``
ogrinfo -sql "update ne_10m_admin_0_countries_points set NAME_LONG = 'North Macedonia' where NAME = 'Macedonia'" osm-lowres.gpkg 
``

Then there is a more complete fix... people might also do GetFeatureInfo on the maps, and there, the name Macedonia would pop up in more attributes and more layers, e..g:

![image](https://user-images.githubusercontent.com/325433/100267057-1c636880-2f53-11eb-9f59-108d3d8d9b45.png)

In particular, found one bring up "Macedonia" with the following queries, present in several attributes:

```
select * from ne_10m_admin_0_countries_points where NAME = 'Macedonia'
select * from ne_10m_admin_1_states_provinces_lines where adm0_name = 'Macedonia'
select * from ne_10m_admin_0_boundary_lines_land where adm0_left = 'Macedonia' or adm0_right = 'Macedonia'
```

If we care about GetFeatureInfo outputs, more complex queries will have to be set up (did not do it yet, as you might not care)

**Comment by simboss** [comment]
@aaime this is enough.

If you had to do the full path, how much time you think you'd need (ballpark estimate).

**Comment by simboss** [comment]
1.5 days in the worst case scenario.

**Comment by simboss** [comment]
@aaime do we have the gpkg somewhere so that we can play with it?

**Comment by simboss** [comment]
@aaime @randomorder I assigned this to @jemacchi as we are starting to use him for EUM as well.

**Comment by jemacchi** [comment]
@simboss @randomorder 
Today started trying to check the change in Dev env, but unfortunatelly the [viewer product](https://view.cgi-dev.de/productviewer?v=default) is working really slowly, showing only the Natural Earth base layer, and no other layer.
**Blocked on testing it.**

On Friday, this is what I did: Found the file for dev env (location, etc). Downloaded. Changed gpkg locally with ogrinfo (since i had it alreadythere). Checked changes with Qgis. Then moved gpkg back (it's on shared folder, renaming original and keeping a backup). After upload, et the end of my day, attempt to test it but it was not possible (then, during weekend I tried again and asked about that, and Alessandro commented that CGI is sending dev servers down to avoid adding cost).
